src/include/mallocMC/mallocMC_utils.hpp

```diff
@@ -100,7 +101,7 @@ namespace CUDA
 
 
 #define warp_serial                                    \
-  for (uint32_richtig_huebsch __mask = __ballot(1),                    \
+  for (uint32_t __mask = __ballot(1),                    \
```

Comment:
I think the header signatures of these CUDA intrinsics are unfortunately [unsigned int](http://docs.nvidia.com/cuda/cuda-c-programming-guide/#warp-vote-functions).

I would therefore either write this explicitly or keep a typedef (with an other name) but not assuming `uint32_t`